src/settingsdialog.cpp

```diff
@@ -32,13 +32,16 @@
 #include <KUrlRequester>
 #include <kconfiggroup.h>
 #include <ksharedconfig.h>
+#include <QKeyEvent>
 #include <QLineEdit>
 #include <QListView>
 
+namespace {
```

Comment:
unrelated change, but a good one - feel free to keep it as-is for now, but in the future try to keep your commits atomic and focused on one thing only

Reply:
This was your first change request

Reply:
now I understand: you squashed this change into the wrong patch, it should be part of the first patch, not the second

Reply:
done is now in the correct commit